projects/angular-showcase/src/app/examples/examples.module.ts

```diff
@@ -44,6 +42,8 @@ import {
   UserMenuModule
 } from '@sbb-esta/angular-public';
 
+import { HeaderModule } from '../../../../sbb-esta/angular-business/src/lib/header/header.module';
```

Comment:
This should be imported from `@sbb-esta/angular-business`

Reply:
Same as above, my bad.